Code/Framework/AzFramework/Platform/Windows/AzFramework/Windowing/NativeWindow_Windows.cpp

```diff
@@ -322,40 +322,40 @@ namespace AzFramework
-            }
-            else
-            {
-                // When running in Windowed mode, we might want either NOTOPMOST or TOPMOST, depending on whether or not a debugger is attached.
-                HWND windowPriority = GetWindowedPriority();
-
-                if (!windowFullScreenState && nativeWindowImpl->m_shouldEnterFullScreenStateOnActivate)
-                {
-                    nativeWindowImpl->m_shouldEnterFullScreenStateOnActivate = false;
-                    nativeWindowImpl->SetFullScreenState(true);
-
-                    // If we're going to fullscreen, then we presumably want to be TOPMOST whether or not a debugger is attached.
-                    windowPriority = HWND_TOPMOST;
-                }
-
-                // When becoming active again, transition from NOTOPMOST to TOPMOST. (Or stay NOTOPMOST if a debugger is attached)
-                SetWindowPos(
-                    nativeWindowImpl->m_win32Handle,
-                    windowPriority, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE | SWP_NOSENDCHANGING);
-            }
+            //const bool windowIsNowInactive = (LOWORD(wParam) == WA_INACTIVE);
+            //const bool windowFullScreenState = nativeWindowImpl->GetFullScreenState();
+
+            //if (windowIsNowInactive)
+            //{
+            //    if (windowFullScreenState)
+            //    {
+            //        nativeWindowImpl->m_shouldEnterFullScreenStateOnActivate = true;
+            //        nativeWindowImpl->SetFullScreenState(false);
+            //    }
+
+            //    // When becoming inactive, transition from TOPMOST to NOTOPMOST.
+            //    SetWindowPos(nativeWindowImpl->m_win32Handle,
+            //        HWND_NOTOPMOST, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE | SWP_NOSENDCHANGING);
+            //}
+            //else
+            //{
+            //    // When running in Windowed mode, we might want either NOTOPMOST or TOPMOST, depending on whether or not a debugger is attached.
+            //    HWND windowPriority = GetWindowedPriority();
+
+            //    if (!windowFullScreenState && nativeWindowImpl->m_shouldEnterFullScreenStateOnActivate)
+            //    {
+            //        nativeWindowImpl->m_shouldEnterFullScreenStateOnActivate = false;
+            //        nativeWindowImpl->SetFullScreenState(true);
+
+            //        // If we're going to fullscreen, then we presumably want to be TOPMOST whether or not a debugger is attached.
+            //        windowPriority = HWND_TOPMOST;
+            //    }
+
+            //    // When becoming active again, transition from NOTOPMOST to TOPMOST. (Or stay NOTOPMOST if a debugger is attached)
+            //    SetWindowPos(
+            //        nativeWindowImpl->m_win32Handle,
+            //        windowPriority, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE | SWP_NOSENDCHANGING);
+            //}
```

Comment:
Should this change get reverted? It looks like something removed locally while debugging.

Reply:
Yes, it need to be reverted.